releasenotes/notes/discrete_signal_bounds_handling-e960ae879c409233.yaml

```diff
@@ -0,0 +1,14 @@
+---
+depreciated:
```

Comment:
As this is not a public attribute this is not a deprecation. You can remove this section.